packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummary.tsx

```diff
@@ -41,17 +42,59 @@ const WholeMeetingSummary = (props: Props) => {
     graphql`
       fragment WholeMeetingSummary_meeting on RetrospectiveMeeting {
         summary
+        reflectionGroups(sortBy: voteCount) {
+          summary
+        }
+        phases {
+          phaseType
+          ... on DiscussPhase {
+            stages {
+              discussion {
+                summary
+              }
+            }
+          }
+        }
         team {
           tier
         }
       }
     `,
     meetingRef
   )
-  const {summary, team} = meeting
-  if (!summary) return null
+  const {summary: wholeMeetingSummary, team, reflectionGroups, phases} = meeting
+  const discussPhase = phases.find((phase) => phase.phaseType === 'discuss')
+  const {stages} = discussPhase ?? {}
   const explainerText =
     team?.tier === 'personal' ? AIExplainer.STARTER : AIExplainer.PREMIUM_MEETING
+  const hasTopicSummary = reflectionGroups.some((group) => group.summary)
+  const hasDiscussionSummary = !!stages?.some((stage) => stage.discussion?.summary)
+  const hasOpenAISummary = hasTopicSummary || hasDiscussionSummary
```

Comment:
It's possible that a topic or discussion summary was generated during the meeting but we've run out of tokens to generate the whole meeting summary, but this is highly unlikely. Worst case scenario, the loading feedback would persist

Reply:
Would this delay the email forever as well?

Reply:
No, the email would send; it would just show a loading spinner